Mockups/Asserts/Assert.debug.cs

```diff
@@ -1,11 +1,50 @@
-﻿namespace IngameScript.Mockups.Asserts
+﻿using System;
+using VRageMath;
+
+namespace IngameScript.Mockups.Asserts
 {
     public static class Assert
     {
-        public static void That(bool condition, string message)
+        [Obsolete("Use " + nameof(Assert) + "." + nameof(True))]
+        public static void That(bool condition, string message) => True(condition, message);
+
+        public static void Fail(string message)
+        {
+            throw new AssertionException(message);
+        }
+
+        public static void True(bool condition, string message)
```

Comment:
These changes goes against my original plan for this file, which was to create an NUnit/FluentAssertions type syntax, but that's OK.

Reply:
I based these off the standard MSTest Asserts, but if you prefer a different language I can change them.